pkg/apis/release/v1alpha1/types.go

```diff
@@ -186,17 +209,17 @@ const (
 // ReleasePayloadCondition contains condition information for a tag event.
 type ReleasePayloadCondition struct {
```

Comment:
Why didn't we use metav1.Condition?

Reply:
Best I can tell, it was only to be able to use custom named Types, which probably isn't necessary.  I've switched to `metav1.Condition` with simple string constants for `Type`.